templates/includes/records/hierarchy-local.html

```diff
@@ -4,32 +4,33 @@
     <div class="container">
         <div class="row">
             <div class="col-4 col-md-2">
-                {% if page.previous_record %}
+                {% if page.previous_record.has_reference_number %}
```

Comment:
I don't think this is right. The Previous and next records should show irrespective of having reference numbers or not.

Reply:
@TNA-Allan I have been told otherwise... we only want to show the links when there is a reference number, as we want what is displayed to be consistent.

Reply:
>@TNA-Allan I have been told otherwise... we only want to show the links when there is a reference number, as we want what is displayed to be consistent.

My concern is about underlying URL and not what is displayed in this context. This condition does not allow to show 'iaid' if `reference number` is not available. Was this thought about?

Reply:
Hi @ababic Example `iaid=C34131` does not have _reference numbers_ in _@next/@previous_. In this case we need the `iaid`.

Reply:
@TNA-Allan I can't find the ticket in Jira any longer, but it's one that was assigned to @jamesbiggs. If I remember rightly, detail on the ticket was pretty light anyway. Not showing the link unless there is a reference number is a detail that came out of a conversation he had with @sdwilkes. You'll just have to trust me on this one.

Reply:
Have reverted the changes to this template... Will submit those changes as a separate PR